src/util/sss_krb5.c

```diff
@@ -367,7 +360,10 @@ krb5_error_code find_principal_in_keytab(krb5_context ctx,
     memset(&cursor, 0, sizeof(cursor));
     kerr = krb5_kt_start_seq_get(ctx, keytab, &cursor);
     if (kerr != 0) {
-        DEBUG(SSSDBG_CRIT_FAILURE, "krb5_kt_start_seq_get failed.\n");
+        const char *krb5_err_msg = sss_krb5_get_error_message(ctx, kerr);
+        DEBUG(SSSDBG_CRIT_FAILURE, "krb5_kt_start_seq_get failed: %s\n",
+              (krb5_err_msg ? krb5_err_msg : ""));
```

Comment:
Maybe it would be better to print something like "- no error message available -" instead of ""?

Reply:
Fixed.